subprojects/composite-builds/src/main/java/org/gradle/composite/internal/RootOfNestedBuildTree.java

```diff
@@ -70,8 +66,6 @@ public RootOfNestedBuildTree(
         super(buildTree, buildDefinition, owner);
         this.buildIdentifier = buildIdentifier;
         this.identityPath = identityPath;
-        this.owner = owner;
-        this.buildName = buildDefinition.getName() == null ? buildIdentifier.getName() : buildDefinition.getName();
```

Comment:
❓ What has the build name been used for? It seems like it used to be the root project name when not assigned separately.

Reply:
It was only used in the `getCurrentPrefixForProjectsInChildBuilds` which was implemented in multiple places in the class hierarchy but not actually used anywhere. You can take a look at the first commit of the PR separately, where the method is removed.